docker-compose.yml

```diff
@@ -12,7 +12,10 @@ services:
     ports:
       - "127.0.0.1:5432:5432"
   mssql:
-    image: mcr.microsoft.com/mssql/server:2017-latest-ubuntu
+  # A working MSSQL server is not available on ARM.
```

Comment:
Isn't there a way to sort of force the architecture? Like throwing `export DOCKER_DEFAULT_PLATFORM=linux/amd64` somwhere?